packages/orchestration/test/facade-durability.test.ts

```diff
@@ -73,7 +73,7 @@ test.serial('chain info', async t => {
   );
 
   const handle = orchestrate('mock', {}, async orc => {
-    return orc.getChain('mock');
+    return orc.getChain('noble');
```

Comment:
I considered adding `mock` to KnownChainNames / KnownChains but it didn't seem worth the effort